manifests/init.pp

```diff
@@ -199,6 +199,8 @@
 #
 # $rails_cache_store::            Set rails cache store
 #
+# $manage_selinux_booleans::      If true AND selinux is enabled on the node, set httpd_can_network_connect so apache works properly
```

Comment:
What about https://github.com/theforeman/puppet-foreman/blob/63bba2150d74670b9cc53d23a4b297b7b10abc32/manifests/config.pp#L157?

The name of this parameter would suggest when setting to false *all* selinux_booleans won't be managed.

Reply:
How should I change it? manage_apache_selinux_boolean? manage_httpd_can_network_connect_boolean?